x/qgb/keeper/keeper_data_commitment.go

```diff
@@ -73,13 +77,13 @@ func (k Keeper) GetLastDataCommitment(ctx sdk.Context) (types.DataCommitment, er
 			return types.DataCommitment{}, err
 		}
 		if !found {
-			return types.DataCommitment{}, fmt.Errorf("couldn't find attestation with nonce %d", latestNonce-i)
+			return types.DataCommitment{}, sdkerrors.Wrapf(types.ErrAttestationNotFound, fmt.Sprintf("couldn't find attestation with nonce %d", latestNonce-i))
 		}
 		dcc, ok := att.(*types.DataCommitment)
 		if !ok {
 			continue
 		}
 		return *dcc, nil
 	}
-	return types.DataCommitment{}, fmt.Errorf("last data commitment not found")
+	return types.DataCommitment{}, sdkerrors.Wrap(types.ErrDataCommitmentNotFound, "last data commitment not found")
```

Comment:
```suggestion
	return types.DataCommitment{}, types.ErrDataCommitmentNotFound
```

I think we don't need to wrap if we're already providing the error that has the same message